### PR TITLE
Fix #1350: Update challengeCreate.js

### LINKED ISSUE #1350: Remove unused code from ChallengeCreate Controller.
**Expected Behaviour:** Remove the commented code after line `84` from controller `ChallengeCreate.js`.
**Hint:** https://github.com/Cloud-CV/EvalAI/blob/master/frontend/src/js/controllers/challengeCreate.js#L84

### PR DESCRIPTION
**Pull request by Ayukha**
Removes unused code from ChallengeCreate Controller. Fixes #1350

### CODE REVIEW

File: frontend/src/js/controllers/challengeCreate.js [hunk 1]
```diff
@@ -80,59 +80,3 @@
         };
     }
 })();
-
```
Comment (RishabhJain2018) [inline frontend/src/js/controllers/challengeCreate.js]: Please don't remove this blank line.

**Comment by Ayukha** [comment]
Done the changes :)  Please review @RishabhJain2018

**Comment by RishabhJain2018** [comment]
Hey @Ayukha Let's motivate @prajwalgatti as he is new to open-source. So, I am accepting his PR. Thanks for your efforts. I would suggest you to take up any other issue from the list :-)

**Comment by Ayukha** [comment]
Sure Sir  :)

**Pull request closed without merging**